feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.html

```diff
@@ -1,5 +1,9 @@
-<ng-container *ngIf="attribute?.values?.length">
```

Comment:
I would the **ng-container** element not remove. This element is not visibale in HTML tree, it only verifies the if-condition. If the statement is true, only then the inner HTML elements will be created/ added to the HTML tree. By means of this statement the HTML tree stays lean

Reply:
an ng-container can't have an id an div can have an id, that's why i replaced the container with an div.
i assumes that the div is not rendered, as well, if the if is false?